plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.tsx

```diff
@@ -74,6 +77,9 @@ const TopologyResourcesTabPanel = ({
             </li>
           ))}
       </TopologyResourcesTabPanelItem>
+      {pipelines?.length ? (
+        <PLRlist pipelines={pipelines} pipelineRuns={pipelineRuns} />
+      ) : null}
```

Comment:
Any issues you see with the above approach? Using ternary operator as it makes the code more readable plus in such scenarios when using the conditional operator in case of `false` it gets converted to 0 and gets rendered that is why adding null instead so that nothing gets rendered. Let me know if you feel otherwise.

Will add a test case for this.

Reply:
it's about preference, am fine either way so can keep it as it is and just add test